Components/ArticleContent.jsx

```diff
@@ -0,0 +1,13 @@
+export default function ArticleContent ({currentArticle}) {
+    return (
+        <>
+            
+            <h2>{currentArticle.title}</h2>
+            {/* <p>🕓 {moment().format(currentArticle.created_at, "DD MM YYYY hh:mm:ss")}</p> */}
```

Comment:
Please get rid of this for now until you get it working 🙂

Reply:
Thanks! I got this working already so no problem. I''ll just fix the other bits, cheers!